app/Integrations/Slack/SlackPlugin.php

```
@@ -0,0 +1,230 @@
+                'action' => 'sent',
+                'value' => 1,
+                'value_unit' => 'message',
+                'event_metadata' => [
+                    'channel' => $event['channel'] ?? null,
+                    'subtype' => $event['subtype'] ?? null,
+                ],
+            ]],
+        ];
+    }
+    
+    protected function convertReactionEvent(array $data, Integration $integration): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$integration'. (Unused Code Rules)

(UnusedFormalParameter)